master/usage/configuration/mtu.md

```diff
@@ -66,42 +66,47 @@ This should either be set on the standalone `libnetwork` service, or on the
 
 ### MTU configuration with CNI
 
-To set the MTU when using CNI the line `"mtu": <MTU size>` must be added to
-the CNI configuration file.
-
-Example CNI configuration
+MTU is set in the by the `"mtu": <MTU size>` field of the CNI configuration. Example:
 
 ```json
-{
+  {
     "name": "any_name",
     "cniVersion": "0.1.0",
     "type": "calico",
     "mtu": 1480,
     "ipam": {
-        "type": "calico-ipam"
+      "type": "calico-ipam"
     }
-}
+  }
 ```
 
-When using the Kubernetes self-hosted manifests, the CNI plugin derives this value from the `veth_mtu`
-field of the calico-config ConfigMap and is set to `1440` by default.
+> **Note**: If using Kubernetes self-hosted manifests, you should modify the
+`veth_mtu` value in the {{site.prodname}} ConfigMap instead, and leave `"mtu"` here
+set to `__CNI_MTU__`. See below for more details.
+{: .alert .alert-info}
+
+### MTU configuration with Kubernetes self-hosted manifests
 
-### Setting MTU for tunnel network interfaces
+When using Kubernetes self-hosted manifests, the CNI plugin derives the MTU value
+from the `veth_mtu` field of the calico-config ConfigMap, and it is set to `1440`
+by default. On restart of the `{{site.nodecontainer}}` pods, any references to
+`__CNI_MTU__` are replaced by the `veth_mtu` value and inserted into the CNI
+configuration file (aka conflist) at the directory specified by Kubernetes
+(currently defaults to `/etc/cni/net.d/`).
 
-If IP-in-IP is enabled and the MTU needs to be modifed then this must be
-configured by setting a Felix environment variable or using calicoctl to set
-the proper configuration variable.  Felix will set the tunnel interfaces to
-the specified MTU.
+Restarting the `{{site.nodecontainer}}` pods will also update any {{site.prodname}}
+tunnel network interfaces on that node. From this point forward, any pods
+started will also have the updated MTU value.
 
-### Setting tunnel MTU through Felix Environment variable
+### Setting tunnel MTU with a Felix environment variable
 
 Passing in the environment variable `FELIX_IPINIPMTU` when running the
 `{{site.nodecontainer}}` container will set the MTU for Felix to use.
 
-When using the Kubernetes self-hosted manifests, the Felix derives this value from the `veth_mtu`
-field of the calico-config ConfigMap and is set to `1440` by default.
+When using the Kubernetes self-hosted manifests, the Felix derives this value from
```

Comment:
I realize that this error was present already, but while you're here, could you remove the incorrect article in front of Felix. It's just Felix, not the Felix. Thanks!